cmd/infra/powervs/create_test.go

```diff
@@ -0,0 +1,31 @@
+package powervs
+
+import (
+	"testing"
+
+	"github.com/IBM-Cloud/power-go-client/power/models"
+)
+
+func TestUseExistingDHCP(t *testing.T) {
```

Comment:
lets have a tabular format

Reply:
I think I have addressed the above comments, ptal!!